### PR TITLE
Potential bug fix for issue #661

### PR DESCRIPTION
**Pull request by K-ran**
This is not a math problem. The onMouseMove method was not called when the mouse was over the "line" (representing the angle). Line being 2px thick, for movements smaller than 2px or any movement over the line, the onMouseMove method was not called. Setting pointer-events of that line to none leads to expected behaviour of the angle picker.  
Only thing is I am not sure this is best way of setting the pointer-event  to none.

### CODE REVIEW

**Comment by K-ran** [comment]
I signed it!

File: core/field_angle.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ goog.inherits(Blockly.FieldAngle, Blockly.FieldTextInput);
  * Round angles to the nearest 15 degrees when using mouse.
  * Set to 0 to disable rounding.
  */
-Blockly.FieldAngle.ROUND = 15;
+Blockly.FieldAngle.ROUND = 1;
```
Comment (rachel-fenichel) [inline core/field_angle.js]: Why did you change this from 15?  I'm guessing it was while debugging, but it should stay at 15 as the default.

File: core/field_angle.js [hunk 2]
```diff
@@ -155,7 +155,8 @@ Blockly.FieldAngle.prototype.showEditor_ = function() {
   this.line_ = Blockly.utils.createSvgElement('line',
       {'x1': Blockly.FieldAngle.HALF,
       'y1': Blockly.FieldAngle.HALF,
-      'class': 'blocklyAngleLine'}, svg);
+      'class': 'blocklyAngleLine',
```
Comment (rachel-fenichel) [inline core/field_angle.js]: CSS changes belong in css.js: https://github.com/google/blockly/blob/master/core/css.js#L452

Comment (K-ran) [inline core/field_angle.js]: Thanks for the review @rachel-fenichel. Just to confirm, should I change the  '.blocklyAngleLine' in the css.js file directly. Is it advisable to do so, because I was afraid it will change its behaviour globally? 
Also after making required changes, I should commit in the same branch and send a pull request, right?
P.S. This is my first pull request.

Comment (rachel-fenichel) [inline core/field_angle.js]: blocklyAngleLine only refers to the line on the angle picker, so changing it in css.js is reasonable.

After making the changes, commit on the same branch locally, then push to your remote branch (K-ran:bugfix/rounding_error).  No need to make a new pull request.

**Comment by K-ran** [comment]
okay, done.
